src/cim/components/featureSupportLevels/ACMFeatureSupportLevelProvider.tsx

```diff
@@ -27,13 +28,19 @@ export const ACMFeatureSupportLevelProvider: React.FC<ACMFeatureSupportLevelProv
     const featureSupportLevelsMap: FeatureSupportLevelsMap = {};
 
     ocpVersions.forEach((version) => {
-      // TODO(mlibra): get this dynamically
-      const featureIdToSupportLevel: FeatureIdToSupportLevel = {
-        REQUESTED_HOSTNAME: 'supported',
-        SNO: 'tech-preview',
-        CNV: 'supported',
-        NETWORK_TYPE_SELECTION: 'tech-preview',
-      };
+      const featureIdParams = {};
+      const featureSupportLevelsJson = JSON.parse(JSON.stringify(featureSupportLevels));
+      featureSupportLevelsJson.supportLevels.forEach(
```

Comment:
It would be good to try to define `getFeatureSupportLevelsMap` function like here and reuse it.  https://github.com/openshift-assisted/assisted-ui-lib/blob/master/src/ocm/components/featureSupportLevels/FeatureSupportLevelProvider.tsx#L29
```suggestion
      getFeatureSupportLevelsMap(featureSupportLevels.supportLevels)
```

Reply:
I don't think we need to map by actual available openshift versions.

Reply:
agree with Jiri - the mapping should be done by the x.y version and not x.y.z, because you can have a cluster with an openshift version x.y.z that isn't one of the available options for a new cluster, causing no support to be returned.
better map between the x.y version and when getFeatureSupportLevel is called, find the x.y version that is the prefix of the version

Reply:
isFeatureSupported needs to be implemented for SNO.
isFeatureDisabled should be fixed to have SNO disabled with the correct tooltip when SNO is unsupported, so it'll be disabled in 4.6

Reply:
ok, i 'll make the changes that you suggested